### ISSUE TITLE
sequali crashes when no adapters are found

### ISSUE THREAD

**Issue by Redmar-van-den-Berg** [rhpvorderman/sequali#159]
Sequali takes the last item from the list of found adapters, which crashes when the list is empty

```bash
Processing TestSample1-R2.fq.gz: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 419k/419k [00:00<00:00, 4.01MiB/s]
Processing TestSample1-R1.fq.gz: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 418k/418k [00:00<00:00, 3.91MiB/s]
Traceback (most recent call last):
  File "/data/micromamba/envs/sequali/bin/sequali", line 11, in <module>
    sys.exit(main())
  File "/data/micromamba/envs/sequali/lib/python3.10/site-packages/sequali/__main__.py", line 281, in main
    report_modules = calculate_stats(
  File "/data/micromamba/envs/sequali/lib/python3.10/site-packages/sequali/report_modules.py", line 2110, in calculate_stats
    AdapterFromOverlapReport.from_insert_size_metrics(insert_size_metrics))
  File "/data/micromamba/envs/sequali/lib/python3.10/site-packages/sequali/report_modules.py", line 1833, in from_insert_size_metrics
    longest_adapter_read1 = adapters_read1[-1][0]
IndexError: list index out of range

```

I've made a small workaround, but I'm not sure exactly where the empty string gets used further on, so use with caution:
```python3
        longest_adapter_read1 = adapters_read1[-1][0] if adapters_read1 else ""
        longest_adapter_read2 = adapters_read2[-1][0] if adapters_read2 else ""
```

The report looks like this with the workaround:
![image](https://github.com/rhpvorderman/sequali/assets/15814544/804c5e84-270c-4759-97aa-b2009350714f)

**Comment by rhpvorderman** [comment]
Thanks for reporting. Let me see if I can create a reproducible test case.

**Comment by Redmar-van-den-Berg** [comment]
I encountered the bug with the `R1.fq.gz` and `R2.fq.gz` files from here: https://github.com/LUMC/HAMLET/tree/main/test/data/fastq

**Comment by rhpvorderman** [comment]
The issue is that it always crashes when there are no adapters. I fixed it in #160
